pkg/retrieval/retrieval_test.go

```diff
@@ -132,3 +132,5 @@ func (s mockPeerSuggester) EachPeer(topology.EachPeerFunc) error {
 func (s mockPeerSuggester) EachPeerRev(f topology.EachPeerFunc) error {
 	return s.eachPeerRevFunc(f)
 }
+
+func FValidate(_ swarm.Chunk) bool { return true }
```

Comment:
why not use the actual validators here if you do actual chunks?

Reply:
I think we used fake validation like this in netstore test as well,, which seems reasonable as we have a separate unit test for validation